src/bun.js/webcore/body.zig

```diff
@@ -1065,8 +1075,12 @@ pub fn BodyMixin(comptime Type: type) type {
             }
 
             var encoder = this.getFormDataEncoding() orelse {
-                globalObject.throw("Invalid MIME type", .{});
-                return .zero;
+                // TODO: catch specific errors from getFormDataEncoding
+                const err = globalObject.createTypeErrorInstance("Can't decode form data from body because of incorrect MIME type/boundary", .{});
```

Comment:
It should be a Promise rejection instead of a thrown error?

Reply:
I think we should return a promise rejection so we can catch the error with the test runner. Same as the errors above for when the body is already used

Reply:
This is a spec question - not a preference for Bun to decide

Browsers do promise rejection so lets be consistent
<img width="362" alt="image" src="https://user-images.githubusercontent.com/709451/226068566-fa05b64c-9a9c-462b-95c6-cd9bf3f17a69.png">

Reply:
This is a fetch spec question - not a preference for Bun to decide.

<img width="362" alt="image" src="https://user-images.githubusercontent.com/709451/226068658-726012d2-0893-4146-a025-31fa11ece7aa.png">

Reply:
👍 switched to throwing the error

Reply:
no 

it should not throw an error

Reply:
sorry